test/src/654-tarball_ingest_special_files/main

```diff
@@ -114,6 +115,33 @@ cvmfs_run_test() {
     echo "*** Found also file from different owner: $file"
   fi
 
+  file=$result_dir/to_empty_link.txt
```

Comment:
I'm not quite sure about the test for the symlinks.  Wouldn't it be natural to have a `readlink` somewhere to test what's the content of the symbolic link?

Reply:
Yes, it is definitely a great enhancement to the test! Thanks for the suggestion :)